workers/unity/Packages/com.improbable.gdk.core/Logging/LogEvent.cs

```diff
@@ -20,14 +26,28 @@ public LogEvent WithField(string key, object value)
             return this;
         }
 
+        public LogEvent WithContext(UnityEngine.Object context)
```

Comment:
Adding another context variable feels cumbersome because there already is the `Data` dictionary, why not use that instead?

Reply:
Discussed offline